pkg/sdn/plugin/pod.go

```diff
@@ -112,6 +118,13 @@ func getIPAMConfig(clusterNetwork *net.IPNet, localSubnet string) ([]byte, error
 					GW: netutils.GenerateDefaultGateway(nodeNet),
 				},
 				{Dst: *clusterNetwork},
+				{
+					// Multicast
+					Dst: net.IPNet{
+						IP:   mcaddr,
+						Mask: net.IPMask(mcaddr),
```

Comment:
This gets the right answer, but not really for the right reason. It would be nicer to use ParseCIDR() on "224.0.0.0/3" to get an IPNet directly.

(And I assume you've tested that this route is actually needed? The default route isn't good enough?)